packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -132,6 +146,11 @@ export class MultiSelectExample extends React.PureComponent<ExampleProps, MultiS
                         tagProps: getTagProps,
                     }}
                     selectedItems={this.state.films}
+                    customTarget={
+                        this.state.customTarget ? (
+                            <MultiSelectCustomTarget count={this.state.films.length} />
+                        ) : undefined
+                    }
```

Comment:
Apologies if this was already discussed, but was there reasons for or against a `renderTarget`-like API that passes in the current list selection to a render function?

The `renderTarget` API prevents mistakes where the state of the list and the data the `customTarget` is reading from can differ.

Reply:
ah good catch, this was not discussed